ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -10,7 +10,7 @@
 from ddtrace.constants import ERROR_TYPE
 from ddtrace.internal.logger import get_logger
 from ddtrace.llmobs import LLMObs
-from ddtrace.llmobs._constants import INPUT_MESSAGES
+from ddtrace.llmobs._constants import INPUT_MESSAGES, INPUT_DOCUMENTS
```

Comment:
Our styling used in the python tracer involves single line imports, ordered alphabetically
```suggestion
from ddtrace.llmobs._constants import INPUT_DOCUMENTS
from ddtrace.llmobs._constants import INPUT_MESSAGES  
```